app/controllers/finance/financial_links_controller.rb

```diff
@@ -13,10 +13,15 @@ def show
       }
     end
     @items += @financial_link.financial_transactions.map do |ft|
+      if ft.group_order
+        ft_note = view_context.link_to ft.note, new_finance_order_path(order_id: ft.group_order.order.id)
+      else
+        ft_note = ft.note
+      end
```

Comment:
I tried to replace this with
`ft_note = render "finance/financial_transactions/order_note", financial_transaction:ft, with_grouporder_link: false`
but this gave me the following error:
`Missing template finance/financial_transactions/order_note with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby, :haml]}. Searched in: * "/app/app/views" * "/app/plugins/wiki/app/views" * "/app/plugins/polls/app/views" * "/app/plugins/messages/app/views" * "/app/plugins/links/app/views" * "/app/plugins/documents/app/views" * "/usr/local/bundle/ruby/2.7.0/gems/twitter-bootstrap-rails-2.2.8/app/views" * "/usr/local/bundle/ruby/2.7.0/gems/kaminari-core-1.2.2/app/views" * "/usr/local/bundle/ruby/2.7.0/gems/doorkeeper-5.6.8/app/views" * "/usr/local/bundle/ruby/2.7.0/gems/actiontext-7.0.8/app/views" * "/usr/local/bundle/ruby/2.7.0/gems/actionmailbox-7.0.8/app/views" Did you mean? finance/financial_transactions/index_collection finance/financial_transactions/new_collection finance/financial_transactions/index finance/balancing/edit_note finance/financial_transactions/new finance/financial_links/incomplete`